pkg/k8s/object/builders/common/ports/instana_agent_port.go

```diff
@@ -60,18 +56,12 @@ func (p InstanaAgentPort) PortNumber() int32 {
 }
 
 func (p InstanaAgentPort) IsEnabled(openTelemetrySettings instanav1.OpenTelemetry) bool {
-	switch p {
-	case OpenTelemetryLegacyPort:
-		fallthrough
-	case OpenTelemetryGRPCPort:
+	if p == OpenTelemetryLegacyPort || p == OpenTelemetryGRPCPort {
 		return openTelemetrySettings.GrpcIsEnabled()
-	case OpenTelemetryHTTPPort:
+	}
+	if p == OpenTelemetryHTTPPort {
 		return openTelemetrySettings.HttpIsEnabled()
-	case AgentAPIsPort:
-		fallthrough
-	case AgentSocketPort:
-		fallthrough
-	default:
-		return true
```

Comment:
Not a behavior change, but I found it less easy to read with switch case, so changed it to multiple if clauses.